mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -11,6 +11,7 @@
 ###########################################################
 
 echo "starting mesos cluster configuration"
```

Comment:
As mentioned in previous comment, I want to hold off on fail fast for now, but let's discuss further offline.